data/json/monsters/monsters.json

```diff
@@ -439,6 +439,7 @@
     "special_attacks": [ [ "scratch", 20 ] ],
     "death_drops": "default_zombie_items",
     "death_function": [ "CONFLAGRATION" ],
+    "upgrades": { "half_life": 14, "into": "mon_fused" },
```

Comment:
It seems weird to have this foe be upgraded from gasoline/gasbag zombies. Because its main ability is rather unrelated and theres no clear evolution line from zombie filled with gas and  a monster that absorb others. 

I would suggest just making them evolve from decayed zombies.

Reply:
The evolution-position is more gameplay related than lore related, since it could be created from every medium-sized zombie. I'd want to place it on tier 3-zombies since tier 1 has some zombies with good health and i find it good to have less tanks on tier 2.  For the lore i mean the gas zombie is gonna adapt at some point to the stored gas and change it's tissue in the process.

Reply:
I think it's best to move this towards evolution from decayed and crawler at a slow rate of 30 half life. Somehow i didn't realise these don't have upgrades.